src/k8s-engine/__tests__/k3sHelper.spec.ts

```diff
@@ -265,4 +266,129 @@ describe(K3sHelper, () => {
       await pendingInit;
     });
   });
+
+  describe('substituting versions', () => {
+    let subject: K3sHelper;
+    let config: settings.Settings;
+
+    beforeAll(() => {
+      config = settings.defaultSettings;
+      subject = new K3sHelper('x86_64');
+      subject['pendingInitialize'] = Promise.resolve();
+    });
+    const convertToSemvers = function(a: [string, string, string]) : [semver.SemVer, semver.SemVer, semver.SemVer] {
+      return [new semver.SemVer(a[0]), new semver.SemVer(a[1]), new semver.SemVer(a[2])];
+    };
```

Comment:
Consider either `function convertToSemvers(a: string, b: string, c: string)` or `function convertToSemvers(...a: string[])` (depending on if you want to enforce exactly three parameters). Either way passing in an array here seems unnecessary.

Also consider `a.map(v => new semver.SemVer(v))`.

Reply:
Implemented that (I think).

Note that Typescript is driving how the variables used with `convertToSemvers` are declared.